cmd/earthly/subcmd/build_cmd.go

```diff
@@ -619,17 +618,27 @@ func (a *Build) ActionBuildImp(cliCtx *cli.Context, flagArgs, nonFlagArgs []stri
 		buildOpts.OnlyArtifactDestPath = destPath
 	}
 
-	// Kick off logstream upload only when we've passed the necessary
+	// Kick off log streaming upload only when we've passed the necessary
 	// information to logbusSetup. This function will be called right at the
 	// beginning of the build within earthfile2llb.
 	buildOpts.MainTargetDetailsFunc = func(d earthfile2llb.TargetDetails) error {
-		a.cli.Console().VerbosePrintf("Logbus: setting organization %q and project %q at %s", d.EarthlyOrgName, d.EarthlyProjectName, time.Now().Format(time.RFC3339Nano))
-		analytics.AddEarthfileProject(d.EarthlyOrgName, d.EarthlyProjectName)
+		// Use the flag/env values for org & project if specified, but fallback to
+		// the PROJECT command if provided.
+		orgName := d.EarthlyOrgName
+		if a.cli.Flags().OrgName != "" {
```

Comment:
should we fallback to the org in the config?
That is what `a.cli.OrgName()` does

Reply:
Thanks! I'll check it out.